man/ronin-web-diff.1.md

```diff
@@ -6,24 +6,28 @@
 
 ## DESCRIPTION
 
-Diffs two separate webpages and prints the HTML nodes which change changed.
+Diffs two separate HTML or XML pages and prints the HTML/XML nodes which got changed.
 
 ## ARGUMENTS
 
 *URL*
   A `https://` or `http://` URL of the web page to diff.
 
 *FILE*
-  A path to the local HTML file to diff.
+  A path to the local HTML or XML file to diff.
 
 ## OPTIONS
 
 `-h`, `--help`
   Print help information
 
+`-f`, `--format`
+  Pass the format of the URL or files. Supported formats are html and xml. (Default: html)
+
 ## AUTHOR
 
 Postmodern <postmodern.mod3@gmail.com>
+Santhosh C @santhosh-chinnasamy
```

Comment:
I don't think contributors should be listed in the `AUTHORS` section. Anyone listed in the `AUTHORS` section should have an email address listed and should be available to answer questions via email about the command and how to use it.

Reply:
removed